tests/webAPI.cpp

```diff
@@ -116,16 +144,42 @@ RetVal makeRequest(const std::string& method, const Params& params)
     return ClientServer::instance().makeRequest<Params, RetVal>(method, params);
 }
 
-bool makeRequest(const std::string& method)
+template <typename RetVal>
+RetVal makeRequest(const std::string& method)
+{
+    return ClientServer::instance().makeRequest<RetVal>(method);
+}
+
+template <typename Params>
+void makeNotification(const std::string& method, const Params& params)
 {
-    return ClientServer::instance().makeRequest(method);
+    ClientServer::instance().makeNotification<Params>(method, params);
+}
+
+void makeNotification(const std::string& method)
+{
+    ClientServer::instance().makeNotification(method);
+}
+
+template <typename T>
+std::shared_ptr<T> addPlugin()
+{
+    return ClientServer::instance().addPlugin<T>();
+}
+
+brayns::Brayns& Brayns()
+{
+    return ClientServer::instance().getBrayns();
 }
 
 BOOST_GLOBAL_FIXTURE(ClientServer);
 
 BOOST_AUTO_TEST_CASE(reset_camera)
 {
-    BOOST_CHECK(makeRequest("reset-camera"));
+    const auto target = Brayns().getEngine().getCamera().getTarget();
+    Brayns().getEngine().getCamera().setTarget({1, 2, 3});
+    makeNotification("reset-camera");
+    BOOST_CHECK_EQUAL(Brayns().getEngine().getCamera().getTarget(), target);
```

Comment:
Brayns() is not a good method name because it shadows the class constructor. Also, it would be more readable to not do 3 times: Brayns().getEngine().getCamera()

Reply:
Will clean this up